test_rclcpp/test/test_timeout_subscriber.cpp

```diff
@@ -40,8 +40,12 @@ TEST(CLASSNAME(test_timeout_subscriber, RMW_IMPLEMENTATION), timeout_subscriber)
   auto start = std::chrono::steady_clock::now();
 
   auto node = rclcpp::Node::make_shared("test_timeout_subscriber");
+
+  rmw_qos_policies_t custom_qos_policies = rmw_qos_policies_default;
```

Comment:
@jacquelinekay yeah, my first attempt did that. But ended up basically replacing the `queue_size` argument with `qos_policy` and didn't make it optional, to mimic the previous behavior that required a queue size to be passed. In any case, I think it's a good idea to have a default QoS policy set so users are not required to pass one every time.